ssqatest/tests/frontend/latha_frontend_tests/test_50percent_coupon.py

```diff
@@ -8,7 +8,7 @@
 import math
 
 
-pytestmark=[pytest.mark.feregression, pytest.mark.fesmoke, pytest.mark.end_to_end]
+pytestmark=[pytest.mark.feregression, pytest.mark.fesmoke]
 @pytest.mark.usefixtures("init_driver")
 

```

Comment:
Do not pus a space between the class name and the decorator. But do put space above the decorator.